cmd/openshift-install/gather.go

```diff
@@ -95,30 +96,30 @@ func runGatherBootstrapCmd(directory string) error {
 	if err != nil {
 		if err2, ok := err.(errUnSupportedGatherPlatform); ok {
 			logrus.Error(err2)
-			return unSupportedPlatformGather()
+			return unSupportedPlatformGather(directory)
 		}
 		return errors.Wrapf(err, "failed to get bootstrap and control plane host addresses from %q", tfStateFilePath)
 	}
 
-	logGatherBootstrap(bootstrap, port, masters)
+	logGatherBootstrap(bootstrap, port, masters, directory)
 	return nil
 }
 
-func logGatherBootstrap(bootstrap string, port int, masters []string) {
-	var (
-		sshport string
-		scpport string
-	)
-	if port != 22 {
-		sshport = fmt.Sprintf(" -p %d", port)
-		scpport = fmt.Sprintf(" -P %d", port)
+func logGatherBootstrap(bootstrap string, port int, masters []string, directory string) {
+	logrus.Info("Pulling logs from bootstrap for debugging")
```

Comment:
nit: reword to "Pulling debug logs from the bootstrap machine" or some such?  Whether folks actually use them for debugging is up to them ;)

Reply:
done https://github.com/openshift/installer/pull/1822/commits/cad7f02a8bdfc268ee54ab4f4e25dec1a99640c3